packages/base-styles/_default-custom-properties.scss

```diff
@@ -1,11 +1,13 @@
+@use "./mixins";
+
 // It is important to include these styles in all built stylesheets.
 // This allows to CSS variables post CSS plugin to generate fallbacks.
 // It also provides default CSS variables for npm package consumers.
 :root {
-	@include admin-scheme(#007cba);
 	--wp-block-synced-color: #7a00df;
 	--wp-block-synced-color--rgb: #{hex-to-rgb(#7a00df)};
```

Comment:
So apparently when there are any functions that can't be resolved, Sass will output the function verbatim, without warning. I can see that this unprefixed `hex-to-rgb` function is being rendered as-is in the built CSS 😅 I understand though, given the amount of official CSS functions being added these days.

There is a [stylelint rule](https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/function-no-unknown/README.md) for this, but we might be disabling it in the repo. We may want to consider enabling that rule for `@wordpress/base-styles` now, and progressively to more packages as we clean them up.

https://github.com/WordPress/gutenberg/blob/d7e0c5d872ba82d851c696cc4c67ea84368a5224/packages/stylelint-config/index.js#L73

Reply:
I tried testing the rule overrides, but unfortunately, it seems that Sass interpolation syntax doesn't warn about functions without a prefix:

![sass-interpolation](https://github.com/user-attachments/assets/111b9a13-eed5-4385-8dee-1903644b1f7a)

This may be an issue with Stylelint itself.

For now, I've fixed it so that adding a prefix will cause it to build correctly. For prefixed functions, a build error will occur if the function does not exist in the namespace, which should prevent misuse.